internal/auth0/organization/flatten.go

```diff
@@ -48,6 +48,8 @@ func flattenOrganizationBranding(organizationBranding *management.OrganizationBr
 func flattenOrganizationConnection(data *schema.ResourceData, orgConn *management.OrganizationConnection) error {
 	result := multierror.Append(
 		data.Set("assign_membership_on_login", orgConn.GetAssignMembershipOnLogin()),
+		data.Set("is_signup_enabled", orgConn.IsSignupEnabled),
```

Comment:
```suggestion
		data.Set("is_signup_enabled", orgConn.GetIsSignupEnabled()),
```

Reply:
We probably want to use the safe getters here and below.

Reply:
With the safe getters, the code doesn't actually work, we need the nil values preserved

Reply:
or at least, that was what happened when I tried earlier, but it is possible I changed too much, I will try again

Reply:
Not sure why, but using the safe getters works fine now. I do agree that it always should have...